R/bundle.R

```diff
@@ -235,7 +235,9 @@ bundleApp <- function(appName, appDir, appFiles, appPrimaryDoc, assetTypeName,
 
   on.exit(setwd(prevDir), add = TRUE)
   bundlePath <- tempfile("rsconnect-bundle", fileext = ".tar.gz")
-  utils::tar(bundlePath, files = ".", compression = "gzip", tar = "internal")
+  tarImplementation <- Sys.getenv("RSCONNECT_TAR", "internal")
+  logger(sprintf("Using tar: %s",tarImplementation))
+  utils::tar(bundlePath, files = ".", compression = "gzip", tar = tarImplementation)
```

Comment:
The documentation for `?tar` has:

> Note that 'files = '.'' will usually not work with an external
> 'tar' as that would expand the list of files after 'tarfile' is
> created.  (It does work with the default internal method.)

We might need to use `getwd()` explicitly here instead, just to be safe? I'm not sure if we have tests already to verify that this does what we want.

Reply:
Hm. Maybe just `files = NULL`, given:

```
files | A character vector of filepaths to be archived: the default is to archive all files under the current directory.
```

Reply:
https://github.com/wch/r-source/blob/8561f62c5d1f464d4a1e491ffc21a42796adf525/src/library/utils/R/tar.R#L352-L359

Reply:
Using `files = NULL` and tested deploys to all the places.